### ISSUE TITLE
Feature Request: Filter/Void Upgrade Existing Items Filter

### ISSUE THREAD

**Issue by belathus** [P3pp3rF1y/SophisticatedStorage#14]
I basically want to be able to set up a void upgrade that keeps one stack of an item and voids the rest -- but does not void items that aren't already in the chest/barrel.

I learned the hard way that the "Void Overflow" option will void things if it is full even if it doesn't have a stack of items already if you set it to Block mode without any items in the filter.

**Comment by P3pp3rF1y** [comment]
I will need details on how to recreate exactly as this works correctly for me - if the item that's being inserted doesn't exist in the barrel and the barrel is full of different items no insert happens whether it is hopper trying to insert something or player shift clicking the stack in it just won't allow that to happen

**Comment by belathus** [comment]
I was using Simple Storage Network to store items. I set up a chest with a void upgrade set to void overflow then filtered a handfull of items. Checked to make sure it wouldn't void anything I didn't want to lose, set the chunk to be chunkloaded, then ran off to explore. Everything that I put into the storage network that didn't have a slot in that chest got voided, and I removed the void upgrade right after.

That said, it did seem to work fine when I tested it initially, so I'll try to recreate it again.
